CHANGES.md

```diff
@@ -4335,7 +4335,7 @@ OpenSSL 1.1.0
 
    *Matt Caswell*
 
- * Excessive allocation of memory in tls_get_message_header() and
+ * Excessive allocation of memory in ssl3_get_message() and
```

Comment:
You changed back the wrong entry in CHANGES.md to `ssl3_get_message`....this entry was correctly referring to `tls_get_message_header()`. You should have changed back the entry on line 15565 :-)

Reply:
Oh I see. Thanks again!